### PR TITLE
updated Newtonsoft.Json, fixed PlaceLimitOrderAsync()

### PR DESCRIPTION
**Pull request by confessore**
ajaja, no power/internet feelsbadman...
working in the service/hospitality industry and keeping up with the local social circle leaves little time for chilling on the net
praying for your satisfaction with whatever your life circumstances are

package update is self explanatory

Good Till Time is a Limit Order Time In Force option

the cancellation of orders does not work as it currently stands

this is fixed by ensuring that the Time In Force: Good Till Time enumeration is applied before specifying the length of time before cancellation

migrating to .NET Standard 2.0 is a much better idea than migrating to .NET Core 2.0
I now realize that Standard is the base for all implementations of .NET including Core

### CODE REVIEW

**Comment by confessore** [comment]
[gdax-csharp.zip](https://github.com/dougdellolio/gdax-csharp/files/1778120/gdax-csharp.zip)

migrating to .NET Standard 2.0 is a much better idea than migrating to .NET Core 2.0.
I now realize that Standard is the base for all implementations of .NET including Core.

File: GDAXClient/Services/Orders/OrdersService.cs [hunk 1]
```diff
@@ -82,6 +82,7 @@ public class OrdersService : AbstractService
             ProductType productPair,
             decimal size,
             decimal price,
+            TimeInForce timeInForce,
```
Comment (dougdellolio) [inline GDAXClient/Services/Orders/OrdersService.cs]: @confessore this fix is super important. can we pull this out into its own pull request and merge immediately? i don't want this mingled in with the net standard changes. Also according to the API, if the user is using `cancel_after`, marking as `GTT` is required. This should be non-optional imo and just a part of the call below on line 96.

**Comment by confessore** [comment]
yeah, I’m out and about right now but I’ll run a clean fork and initiate a
pull request as soon as I make it back
